### PR TITLE
Update lineage service to use versioning functionality

### PR DESCRIPTION
**Pull request by conradoverta**
This adds small changes to our existing lineage service to drop the existing data versioning solution in favor of the commit-based one.

### CODE REVIEW

**Review by evgeniilezhnin — APPROVED** [review]

**Comment by conradoverta** [comment]
@evgeniilezhnin holding off on this change to avoid breaking the build. You can place your code on top of it.

**Comment by evgeniilezhnin** [comment]
why this new id is required? How can we know this id when we want to delete something? 

I mean it is returned only when we add lineage, but not when we get existing lineage.

File: protos/protos/public/modeldb/Lineage.proto [hunk 1]
```diff
@@ -39,18 +35,27 @@ message LineageEntryBatch {
...
-        bool status = 1;
+        uint64 id = 1;
     }
 }
 
 // Delete every item of "input" as a parent of every item on "output".
 message DeleteLineage {
-    repeated LineageEntry input = 1;
-    repeated LineageEntry output = 2;
+    uint64 id = 1; // Required
```
Comment (ravishetye) [inline protos/protos/public/modeldb/Lineage.proto:51]: is the design that we can partially delete a Lineage entry? if not then just the id should be sufficient, right?

**Review by ravishetye — APPROVED** [review]

File: protos/protos/public/modeldb/Lineage.proto (hunk 1 above)
Comment (conradoverta) [reply protos/protos/public/modeldb/Lineage.proto:51]: Yes, it should cover partial delete.

**Pull request closed without merging**